flowlogd/put_flow_logs.py

```diff
@@ -130,3 +130,20 @@ def get_log_enable_account_ids():
 
     res = jclient.vpc.describe_flow_log_enable_accounts('describe-flow-log-enable-accounts')
     return res['DescribeFlowLogEnableAccountsResponse']['accountIds']['item']
+
+def delete_flows_objects(bucket_name):
+    LOG.info('Deleting for %s logs older than 7 Days' % bucket_name)
```

Comment:
7 days ---> days you can calculate from periodic_purge_task_interval flag in the flowlogd.cfg